types/tsconfig.json

```diff
@@ -3,9 +3,11 @@
     "target": "ESNext",
     "moduleResolution": "node",
     "strict": true,
+    "noImplicitAny": false,
```

Comment:
There are a ton of errors produced be implicit any for each RPC handler argument.
Example:
```
application/api/auth.2/register.js:3:20 - error TS7031: Binding element 'login' implicitly has an 'any' type.

3   method: async ({ login, password, fullName }) => {
                     ~~~~~

application/api/auth.2/register.js:3:27 - error TS7031: Binding element 'password' implicitly has an 'any' type.

3   method: async ({ login, password, fullName }) => {
                            ~~~~~~~~

application/api/auth.2/register.js:3:37 - error TS7031: Binding element 'fullName' implicitly has an 'any' type.

3   method: async ({ login, password, fullName }) => {
```